pmm-tests/pmm-framework.sh

```diff
@@ -36,6 +36,7 @@ usage () {
   echo " --ms-version                     Pass MySQL Server version info"
   echo " --md-version                     Pass MariaDB Server version info"
   echo " --pxc-version                    Pass Percona XtraDB Cluster version info"
+  echo " --mysqld-startup-options         Pass MySQL startup options"
```

Comment:
Maybe it will be great to add hint on option --help how to pass options - "--innodb_buffer_pool_size=1G --innodb_log_file_size=1G"